src/hdmf/term_set.py

```diff
@@ -162,19 +162,21 @@ def __schemasheets_convert(self):
         This method returns a path to the new schema to be viewed via SchemaView.
         """
         try:
-            import yaml
+            from ruamel.yaml import YAML
```

Comment:
This can be moved to the top of the file.

Reply:
I think it should stay because YAML is not required for TermSet. It's only needed for schemasheets which is one method.